### PR TITLE
Add NORDic recipe

### PR DESCRIPTION
**Pull request by clreda**
Adds package [NORDic](https://github.com/clreda/NORDic) which builds and analyzes disease-specific gene regulatory networks (as Boolean networks) for drug repurposing.

### CODE REVIEW

File: recipes/nordic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - omnipath >=1.0.6
+    - openpyxl >=3.0.10
+    - pandas >=1.5.1
+    - qnorm >=0.5.1
+    - quadprog >=0.1.11
+    - scipy >=1.6.2
+    - seaborn >=0.12.1
+    - tqdm >=4.62.3
+  run:
+    - python >=3.8.0,<4.0.0
```
Comment (mencian) [inline recipes/nordic/meta.yaml:44]:
```suggestion
build:
  number: 0
  skip: True  # [py < 38]
  run_exports:
    - {{ pin_subpackage(name, max_pin="x") }}

requirements:
  build:
    - {{ compiler('c') }}
    - make
  host:
    - python
    - pip
    - clingo >=5.6.1
    - cmappy
    - scikit-learn
    - graphviz
    - matplotlib-base
    - numpy >=1.22.4
    - omnipath >=1.0.6
    - openpyxl >=3.0.10
    - pandas >=1.5.1
    - qnorm >=0.5.1
    - quadprog >=0.1.11
    - scipy >=1.6.2
    - seaborn >=0.12.1
    - tqdm >=4.62.3
  run:
    - python
```

File: recipes/nordic/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+cd "${SRC_DIR}/MaBoSS-2.0/src"
+make install \
+    CC="${CC}" \
+    CXX="${CXX}" \
+    CFLAGS="${CFLAGS}" \
+    CXXFLAGS="${CXXFLAGS}" \
+    LDFLAGS="${LDFLAGS}" \
+    prefix="${PREFIX}"
+"${PYTHON}" -m pip install bonesis maboss mpbn
+"${PYTHON}" -m pip install --no-deps --ignore-installed . -vv
```
Comment (mencian) [inline recipes/nordic/build.sh:12]:
```suggestion
"${PYTHON}" -m pip install --no-deps --no-build-isolation . -vvv
```

**Comment by clreda** [comment]
@BiocondaBot please add label

**Comment by mencian** [comment]
Thanks, @clreda! This is a neat workaround to get the packages not yet in Bioconda installed.

**Review by mencian — APPROVED** [review]

**Pull request merged**